core/arch/arm/kernel/vfp.c

```diff
@@ -32,9 +32,18 @@ void vfp_lazy_save_state_init(struct vfp_state *state)
 	vfp_write_fpexc(fpexc & ~FPEXC_EN);
 }
 
+static bool force_save(struct vfp_state *state)
+{
+#ifdef CFG_WITH_ARM_TRUSTED_FW
+	return state->force_save;
+#else
+	return false;
+#endif
+}
```

Comment:
OK, but how about making things a bit simpler by always having `force_save` in `struct vfp_state`? Saving one bool on non-TF-A platforms is not a big deal, is it?

Reply:
It will expand to 16 bytes per core. I could perhaps move it into `struct thread_vfp_state` instead where it would fit in some unused padding.